bukkit/src/main/java/co/aikar/commands/BukkitCommandManager.java

```diff
@@ -403,4 +414,19 @@ protected boolean handleUncaughtException(BaseCommand scope, RegisteredCommand r
         }
         return super.handleUncaughtException(scope, registeredCommand, sender, args, t);
     }
+
+    @Override
+    public void enableUnstableAPI(String api) {
+        super.enableUnstableAPI(api);
+
+        if ("adventure".equals(api) && adventureAvailable) {
+            log(LogLevel.INFO, "Enabling Adventure");
```

Comment:
not sure a message here is necessary, ppl already complain about the current messages...